docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,297 @@
+window.AddAvailableOrientation( Portrait );
+
+// To enable Landscape
+window.AddAvailableOrientation( Landscape );
+
+// To disable PortraitInverse
+window.RemoveAvailableOrientation( PortraitInverse );
+
+// To enable LandscapeInverse
+window.AddAvailableOrientation( LandscapeInverse );
+```
+
+
+### Preferred orientation
+
+If you want the window to appear in a specific orientation, the preferred orientation function is supported. The preferred orientation is one of the available orientations. For example, as shown in the following code, the available orientation list has `Portrait`, `Landscape`, and `LandscapeInverse`, and the preferred orientation is set to Landscape. This implies that the window will appear in landscape even when the device is rotated with any other orientation.
+
+```csharp
+// To enable available orientations
+window.AddAvailableOrientation( Portrait );
+window.AddAvailableOrientation( Landscape );
+window.AddAvailableOrientation( LandscapeInverse );
+
+// To set the preferred orientation with Landscape.
+window.SetPreferredOrientation( Landscape );
+```                                                                                                                               
+> [!NOTE]
+>
+> If the Preferred Orientation function is used without listing the supported orientations (`AddAvailableOrientation`), the rotation will not work as expected.
+>
+
+## Multi-Window
+MultiWIndow means that application use additional window except default window.
+
+<img src="./media/multiwindow3.png">
+
+To Create MultiWindow, the device must support the following features:
+- https://www.tizen.org/feature/opengles.surfaceless_context
+
+### Create additional window
+
+It is easy to create an additional window in addition to the default window. The size and position of the new window should be specified. Each window can have its own background color and title:
+
+```csharp
+Window newWindow = new Window(new Rectangle(0, 0, 1920, 1080))
+{
+    BackgroundColor = Color.White,
+    Title = "new window"
+};
+```
+
+> **Note**
+>
+> To support rotation in multi window, the [rotation](#rotation) functions should be used for each window.
+>
+
+## Change window stacking order
+
+NUI provides support for changing the stacking order of the windows.
+
+A window can be raised to the top of the window stack so that no sibling window obscures it:
+
+```csharp
+window.Raise();
+```
+
+A window can be lowered to the bottom of the window stack so that it does not obscure any sibling windows:
+
+```csharp
+window.Lower();
+```
+
+New windows are automatically placed to the top of the window stack at creation time by the window manager.
+
+
+### Set parent window
+
+To set transient relationship between two windows, use `SetParent()`. After setting the relationship, the two windows together raise-up, lower, and iconified or deiconified. Initially, the child window is located on top of the parent. The window can go below parent by calling `Lower()`:
```

Comment:
@surbhig12 Would you please check this sentence again? To me below format seems more proper.

... the two windows are together raised-up, lowered, and iconified or deiconified....

Reply:
@mijin85cho, I agree.
@tscholb,

Suggest changing to:

To set transient relationship between two windows, use `SetParent()`. After setting the relationship, the two windows are together raised-up, lowered, and iconified or deiconified. Initially, the child window is located on top of the parent. The window can go below parent by calling `Lower()`: